### ISSUE TITLE
Propose: Form to help PR

### ISSUE THREAD

**Issue by joubertredrat** [Kickball/awesome-selfhosted#282]
Hi @Kickball and @nodiscc 
I'm looking there that although it has a syntax to make PR, a lot of applications on list isn't on this syntax.

Few days ago @nodiscc asked me to make a PR and I'm even started, but when I saw a lot of applications that I put on issues, I saw that would be very boring format all, then this form can help in this case too.

I made a simple alpha for testing in https://joubertredrat.github.io/awesome-selfhosted-form/

If you like this, I can format page and make PR or host in another way.

That you think, liked this? :D

**Comment by nodiscc** [comment]
> a lot of applications on list isn't on this syntax.

Yes, see https://github.com/Kickball/awesome-selfhosted/issues/35 and #13 

>  I saw that would be very boring format all,

Manually formatting markdown or using your tool are equally boring for me :) However it costs nothing to add "You can also use https://joubertredrat.github.io/awesome-selfhosted-form/ which will take care of the formatting for you." to the _Contributing_ section. If @Kickball does agree, we can do that. 

You may also want to add a leading ` * ` (space-asterisk-space) to your tool output and some usage instructions.

**Comment by Kickball** [comment]
Great tool, thanks for putting in the work to make it!

I'm getting the following error in Chrome Version 45.0.2454.99 m

    Refused to execute script from 'https://raw.githubusercontent.com/joubertredrat/bootstrap-maxlength/master/src/bootstrap-maxlength.js' because its MIME type ('text/plain') is not executable, and strict MIME type checking is enabled.
    joubertredrat.github.io/:25 Uncaught TypeError: $(...).maxlength is not a function

I don't think I've locked down my browser any more than normal.

Once we can find a solution to that bug then we can put a link to it on the main page. :smile:

**Comment by joubertredrat** [comment]
Hi,
@Kickball Okay, I will fix this issue and format page and input then.

**Comment by joubertredrat** [comment]
Hi, 
@Kickball @nodiscc It's done! Look a beta and talk me if you like.
https://joubertredrat.github.io/awesome-selfhosted-form/

License select now have options
Language select now have options but you can write project language if not on list.
Various links to https://github.com/Kickball/awesome-selfhosted/ repo

**Comment by Kickball** [comment]
Looks good. I think I'll put it on the repo now, and add a link to the contributing section.

Moving forward though, could we adapt it so it can deal with the paid/non-free self-hosted software?

**Comment by joubertredrat** [comment]
Hi guys,
@Kickball yes, is possible, you like another form on same page or another page for paid/non-free self-hosted form?

**Comment by Kickball** [comment]
Another tab on the same page would be the best option in my opinion.

Please build the system so that we can link directly to the non-free tab, eg:

https://joubertredrat.github.io/awesome-selfhosted-form#non-free

**Comment by nodiscc** [comment]
@joubertredrat just a few remarks about https://joubertredrat.github.io/awesome-selfhosted-form/

 * I'd prefer if it didn't do cross-site requests to jquery.com and cloudflare.com
 * License list doesn't work right now.
 * About non-free items, I think the easiest way to do it would be adding a `Non-free` item to the license list, if this is selected, add a reminder above the markdown field: _You have selected a non-free license, this entry should be added to [non-free.md](https://github.com/Kickball/awesome-selfhosted/blob/master/non-free.md)_. No confusion possible this way. @Kickball  What do you think?

**Comment by joubertredrat** [comment]
Hi guys,

Okay @nodiscc I will fix this issues. Sorry for the delay, on this week I'm preparing a lecture for an event in Brazil, "Linux week". My lecture is about Openstack.

**Comment by nodiscc** [comment]
Thanks a lot, no hurry.

**Comment by joubertredrat** [comment]
@nodiscc 
* I'd prefer if it didn't do cross-site requests to jquery.com and cloudflare.com = fixed
* License list doesn't work right now. = I can't produce this problem here, what browser you using? You use adblock, ghostery or another js tool?
* About non-free items, I think the easiest way to do it would be adding a Non-free item to the license list, if this is selected, add a reminder above the markdown field: You have selected a non-free license, this entry should be added to non-free.md. No confusion possible this way. = I add a "Non-free" on licences.

you can test now :)

**Comment by nodiscc** [comment]
Everything looks ok except:

 * `⚠` should be a checkbox, not in licenses list
 * Missing message `You have selected a non-free license, this entry should be added to non-free.md`

**Comment by joubertredrat** [comment]
@nodiscc @Kickball 
New feature, look a small button "Open Issue form with this"

**Comment by nodiscc** [comment]
Nice work. Added in https://github.com/Kickball/awesome-selfhosted/commit/749d7f65bb0116a69c8bad6a5d15932087dbdba6, thanks a lot @joubertredrat. Let me know if something should be changed.

**Comment by Kickball** [comment]
@joubertredrat Great work! I love the open issue button, its an amazing feature.

Are we still moving forward with @nodiscc 's suggestion of a `⚠ - Depends on a third party network service` checkbox?

**Comment by Kickball** [comment]
@joubertredrat 

Are we still moving forward with @nodiscc 's suggestion of a `⚠ - Depends on a third party network service` checkbox?

**Comment by joubertredrat** [comment]
Done
